trl/extras/vllm_client.py

```diff
@@ -231,7 +234,7 @@ def update_named_param(self, name: str, weights: torch.Tensor):
             raise Exception(f"Request failed: {response.status_code}, {response.text}")
 
         # Broadcast the weights to the other processes
-        self.pynccl_comm.broadcast(weights, src=self.rank, stream=torch.cuda.current_stream())
```

Comment:
stream isn't needed, right? it defaults to the current stream?

Reply:
Ok I got my answer: https://github.com/huggingface/trl/pull/3286/files#r2040619510